packages/kit/src/runtime/client/client.js

```diff
@@ -677,6 +677,18 @@ export function create_client({ target, base }) {
 				try {
 					lock_fetch();
 					data = (await node.shared.load.call(null, load_input)) ?? null;
+					const not_object = typeof data !== 'object' && data != null;
+					if (not_object || Array.isArray(data) || data instanceof Response) {
+						throw new Error(
+							`a load function related to route '${route.id}' returned ${
+								not_object
+									? `a ${typeof data}`
+									: data instanceof Response
+									? 'a Response object'
+									: 'an array'
+							}, but must return a plain object (e.g '{ data: ...}')`
```

Comment:
A bit of a nitpick, but this would result in the data being accessing in the page via `data.data`, which we shouldn't encourage. Maybe suggest a non-data word instead?

```suggestion
							}, but must return a plain object (e.g '{ values: ... }')`
```

Reply:
`values` makes sense as a suggestion if what they were returning was an array, but I'm not sure it does otherwise.

Reply:
Ah yeah, I guess I'm thinking about this in the array context since that's been a common source of confusion.

Maybe ` { stuff: ... }` ?

Reply:
I went with `{ something: .. }`